src/Stratis.Bitcoin.Features.Wallet.Tests/WalletTransactionHandlerTest.cs

```diff
@@ -318,10 +319,13 @@ public void FundTransaction_Given__a_wallet_has_enough_inputs__When__adding_inpu
             Assert.Single(fundTransaction.Inputs); // 4 inputs
 
             Transaction fundTransactionClone = this.Network.CreateTransaction(fundTransaction.ToBytes());
-            var fundContext = new TransactionBuildContext(this.Network, walletReference, new List<Recipient>(), "password")
+            var fundContext = new TransactionBuildContext(this.Network)
             {
+                AccountReference = walletReference,
                 MinConfirmations = 0,
-                FeeType = FeeType.Low
+                FeeType = FeeType.Low,
+                WalletPassword = "password",
+                Recipients = new List<Recipient>()
```

Comment:
Can't this at least be in the constructor as `new List<Recipient>()`? Instead of having to define it here like this even if its an empty list?

Reply:
I moved them all out of the constructor so that the consumer doesn't wonder why this one is in the ctor and not the others.
So that they're all at the same level if that makes sense.